Documentation~/Cinemachine3rdPersonFollow.md

```diff
@@ -0,0 +1,75 @@
+# 3rd person follow
+
+Use Cinemachine Virtual Camera’s 3rd Person Follow to keep the camera at a constant position and distance relative to the Follow target (subject to damping controls), tracking the target’s movement and rotation.
+
+The virtual camera is bolted to a virtual rig that is attached to the target. The red rods and dots visible around the target in the scene view reflect the rig setup as well as the camera position and distance relative to the target.
+
+The rig is defined by three points: the **Shoulder Offset**, the **Vertical Arm Length**, and the **Camera Distance**.
```

Comment:
**Vertical Arm Length** and **Camera Distance** are not points

Reply:
Done.